src/Control/Monad/Ghc.hs

```diff
@@ -25,12 +27,22 @@ instance (Functor m, Monad m) => Applicative (GhcT m) where
   pure  = return
   (<*>) = ap
 
+-- adapted from https://github.com/ghc/ghc/blob/59fa7b32b018a91f81773ca676251a0b2761ef56/compiler/main/GHC.hs#L454-L463
+-- modified to _not_ catch ^C
+rawRunGhcT :: (MonadIO m, MonadMask m) => Maybe FilePath -> GHC.GhcT (MTLAdapter m) a -> MTLAdapter m a
+rawRunGhcT mb_top_dir ghct = do
+  ref <- liftIO $ newIORef (error "empty session")
+  let session = GHC.Session ref
+  flip GHC.unGhcT session $ {-GHC.withSignalHandlers $-} do -- do _not_ catch ^C
```

Comment:
I assume `{-GHC.withSignalHandlers $-}` is to make it very clear what our modification is?

Reply:
yes